opmon/android-add-ons-extension-process.toml

```diff
@@ -0,0 +1,14 @@
+[project]
+
+name = "Android Add Ons Extension Process"
+platform = "fenix"
+metrics = [
+  "storage_stats_app_bytes",
+  "storage_stats_cache_bytes",
+  "storage_stats_data_dir_bytes"
```

Comment:
These are some of the memory related metrics available on Fenix and that are defined here: https://github.com/mozilla/metric-hub/blob/972480c8fa975331f3edf625a2e68d5e63581861/definitions/fenix.toml#L192-L214

Feel free to add more metrics. There are docs available here: https://docs.telemetry.mozilla.org/cookbooks/operational_monitoring.html